README.md

```diff
@@ -29,6 +29,7 @@ added and modified in the AMBF Yaml config files.
 
 #### Notes:
 1. This plugin is only supported for **Blender 2.79**
+2. Simulation frame must be at 0 (box to the right of the start end frames)
```

Comment:
Maybe we can write that "In the Blender Timeline Toolbar which is meant for controlling the animation time, the current time frame should be set to 0 seconds. The current time frame is shown as a box to the right of the start end frames. It is 0 by default, but accidentally pressing some keyboard keys may step this time forward."